packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.stories.ts

```diff
@@ -43,7 +48,7 @@ type Story = StoryObj<typeof OnyxTooltip>;
 export const Default = {
   args: {
     text: "Tooltip text",
-    default: () => "Slot content goes here",
+    default: () => h("span", { tabindex: "0" }, "Slot content goes here"),
```

Comment:
See comment before, we should avoid promoting setting `tabindex: 0`.

Reply:
I used a button for all examples now